src/app/example.component.html

```diff
@@ -48,6 +48,17 @@ <h3>Layers</h3>
     <fa-layers-text title="Fort Awesome" [content]="'Fa'" transform="up-4" [styles]="{'font-size': '0.3em'}"></fa-layers-text>
   </fa-layers>
 </div>
+<p>Increase the size of the layers container.</p>
+<div class="fa-4x">
```

Comment:
@devoto13 - thanks for noticing this, it is a left over class which I have corrected.  I also corrected all the other examples that are using `fa-layers` to use the new attribute instead of setting `[ngClass]="['fa-fw']"` manually so that all the examples are consistent.

Reply:
Great, thanks for updating it! 
Can you also replace `class="fa-4x"` with `size="4x"` on `fa-layers` element, so it uses the newly added property?

Reply:
@devoto13 - I have implemented the requested changes.  Also, I removed the section I had initially added to the examples since the use-case and use of both `[fixedWidth]` and `size` was already demonstrated in the existing examples after making this change.

Also, our implementation on this (now and the prior implementation) does not allow custom CSS classes to be defined by the user (e.x. a user cannot add `class="foo"` or `[ng-class]="bar"` directly to the `fa-layers` component because we are overriding the CSS in our component.  Do we want to call this out anywhere in the documentation that any additional classes must be on some wrapper element?